cli/src/main/java/org/jboss/pnc/bacon/cli/App.java

```diff
@@ -103,9 +104,8 @@ public static void main(String[] args) throws Exception {
     }
 
     private static void initializeConfig() {
-        String configLocation = System.getProperty("config", "config.yaml");
         try {
-            Config.configure(configLocation);
+            Config.configure(Constant.DEFAULT_CONFIG_FOLDER, Constant.CONFIG_FILE_NAME);
```

Comment:
I would maybe move this to the `setConfigurationFileLocationIfPresent` method so all three ways are at one place (and then renamed the method just `setConfigurationFileLocation`)